tokens/finn.no/buttons.yml

```diff
@@ -4,97 +4,97 @@ token: maps
 color:
   button:
     primary:
-      text: white
+      text: s-color-text-inverted
       background:
-        _: blue-600
-        hover: blue-700
-        active: blue-800
+        _: s-color-background-primary
+        hover: s-color-background-primary-hover
+        active: s-color-background-primary-active
       border:
-        _: blue-600
-        hover: blue-700
-        active: blue-800
-      icon: white
+        _: s-color-border-primary
+        hover: s-color-border-primary-hover
+        active: s-color-border-primary-active
+      icon: s-color-icon-inverted
     secondary:
-      text: blue-600
+      text: s-color-text-link
       background:
-        _: white
-        hover: blue-50
-        active: blue-200
+        _: s-color-background
+        hover: s-color-background-hover
+        active: s-color-background-active
       border:
-        _: bluegray-300
-        hover: blue-700
-        active: blue-800
-      icon: blue-600
+        _: s-color-border
+        hover: s-color-border-hover
+        active: s-color-border-active
+      icon: s-color-icon-primary
     quiet:
-      text: blue-600
+      text: s-color-text-link
       background:
         _: transparent
-        hover: gray-100
-        active: gray-200
-      icon: blue-600
+        hover: s-color-background-hover
+        active: s-color-background-active
+      icon: s-color-icon-primary
     negative:
-      text: white
+      text: s-color-text-inverted
       background:
-        _: red-600
-        hover: red-700
-        active: red-800
+        _: s-color-background-negative
+        hover: s-color-background-negative-hover
+        active: s-color-background-negative-active
       border:
-        _: red-600
-        hover: red-700
-        active: red-800
-      icon: white
+        _: s-color-border-negative
+        hover: s-color-border-negative-hover
+        active: s-color-border-negative-active
+      icon: s-color-icon-inverted
       quiet:
-        text: red-600
+        text: s-color-text-negative
         background:
           _: transparent
-          hover: red-100
-          active: red-200
+          hover: s-color-background-negative-subtle-hover
+          active: s-color-background-negative-subtle-active
         border:
           _: transparent
           hover: transparent
           active: transparent
-        icon: red-600
+        icon: s-color-icon-negative
     disabled:
-      text: white
-      background: bluegray-300
+      text: s-color-text-inverted
+      background: s-color-background-disabled
       quiet:
-        text: bluegray-300
-        border: bluegray-300
+        text: s-color-text-link-disabled
+        border: s-color-border-disabled
     link:
       text: s-color-text-link
     pill:
       background:
         _: transparent
-        hover: blue-300-transparentized-30-percent
+        hover: blue-300-alpha30
```

Comment:
It feels like there might be a pill-background-active token missing here to indicate that the pill button was pressed. That's at least how other types of buttons are built. I added a [comment to the spreadsheet](https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=1588196525&range=E123).